src/nacl/pwhash.py

```diff
@@ -239,5 +341,8 @@ def verify_argon2i(password_hash, password):
 
     .. versionadded:: 1.2
     """
-    return nacl.bindings.crypto_pwhash_argon2i_str_verify(password_hash,
-                                                          password)
+    return nacl.bindings.crypto_pwhash_str_verify(password_hash,
+                                                  password)
+
+
+verify_argon2i = verify_argon2
```

Comment:
Did we do a release with argon2i exposed? If not we don't need this.

Reply:
Never released; removed in 6eda5e4